packages/r/rust/xmake.lua

```diff
@@ -0,0 +1,35 @@
+package("rust")
+    set_kind("toolchain")
+    set_homepage("https://rust-lang.org")
+
+    if is_host("windows") then
+        add_urls("https://static.rust-lang.org/rustup/archive/$(version)/x86_64-pc-windows-msvc/rustup-init.exe")
+
+        add_versions("1.28.1", "7b83039a1b9305b0c50f23b2e2f03319b8d7859b28106e49ba82c06d81289df6")
+    end
+
+    if is_host("linux") then
+        add_extsources("apt::rustup")
+        add_extsources("pacman::rustup")
+    elseif is_host("macosx") then
+        add_extsources("brew::rustup")
+    end
+
+    on_install(function (package)
```

Comment:
```suggestion
    on_install("@windows|x64", function (package)
```
Since download is x86_64.